foxy/distribution.yaml

```
@@ -2908,8 +2908,8 @@ repositories:
       - tile_map
       tags:
         release: release/foxy/{package}/{version}
-      url: https://github.com/swri-robotics-gbp/mapviz-release.git
-      version: 2.1.0-1
+      url: git@github.com:ros2-gbp/mapviz-release.git
```

We shouldn't force consumers to use SSH to clone release repositories. I'm pretty sure the buildfarm can't do it anyway.

```suggestion
      url: https://github.com/ros2-gbp/mapviz-release.git
```